rdtools/__init__.py

```diff
@@ -1,6 +1,9 @@
 from rdtools.normalization import normalize_with_sapm
 from rdtools.normalization import normalize_with_pvwatts
 from rdtools.normalization import irradiance_rescale
```

Comment:
I didn't realize `rdtools` was explicitly importing so many submodules. Generally I prefer leaving the python namespace clear and using local aliasing if needed.

Reply:
Agreed. But putting them in `__init__.py` only makes them available as, e.g. `rdtools.normalize_with_sapm()` it prevents the user from needing to use `rdtools.normalization.normalize_with_sapm()`. They don't end up polluting the python namespace. Did I understand your comment correctly?

```python
In [1]: import rdtools

In [2]: rdtools.normalize_with_sapm
Out[2]: <function rdtools.normalization.normalize_with_sapm>

In [3]: normalize_with_sapm
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-3-1103d42db088> in <module>()
----> 1 normalize_with_sapm

NameError: name 'normalize_with_sapm' is not defined
```

Reply:
OK, I understand now.  Looks good.